src/ActivityLogger.php

```diff
@@ -103,7 +103,7 @@ public function by($modelOrId)
      */
     public function withProperties($properties)
     {
-        $this->properties = collect($properties);
+        $this->properties = $this->properties->merge(collect($properties));
```

Comment:
`merge()` could be called with an array and everything else a collection can handle - so far I know.

Reply:
Thanks, will look into it :)